src/storage/validators/x-robots-tag.ts

```diff
@@ -0,0 +1,188 @@
+import { ERRORS } from '@internal/errors'
+
+const SIMPLE_RULES = [
```

Comment:
Ignore me if this is already discussed but otherwise wanted let you know that `nocache`/`noarchive` seem still in use by some (i.e. [Bing](https://www.bing.com/webmasters/help/which-robots-metatags-does-bing-support-5198d240)) and so excluding them can fail normally valid requests in validation